site/using-weave/awsvpc.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Using IP Routing on an Amazon Web Services Virtual Private Cloud
+menu_order: 110
+---
+
+If you are running your container infrastructure entirely within
+Amazon Web Services (AWS) Elastic Compute Cloud (EC2), then you can
+choose AWS-VPC mode to connect your containers without any overlay, so
+they can operate very close to the full speed of the underlying
+network.
+
+In this mode, Weave Net still manages IP addresses and connects
+containers to the network, but instead of wrapping up each packet and
+sending it to its destination, Weave Net instructs the AWS network
+router which ranges of container IP addresses live on which instance.
+
+![Weave Net AWS-VPC Mode](weave-net-awsvpc-1007x438.png)
+
+###Configuring your EC2 instances to use Weave AWS-VPC mode
+
+First, your AWS instances need to be given access to change the route
+table.  Give them an IAM Role which has this access; if you have an
+existing role then extend it or create a new one with a policy
+allowing the necessary actions:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:CreateRoute",
+                "ec2:DeleteRoute",
+                "ec2:ReplaceRoute",
+                "ec2:DescribeRouteTables",
+                "ec2:DescribeInstances"
+            ],
+            "Resource": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+Your Security Group must allow network traffic between instances: you
+must open TCP port 6783 which Weave Net uses to manage the network,
+and you must allow any ports which your own containers use. Remember:
+there is no network overlay in this mode so IP packets with container
+addresses will flow over the AWS network unmodified.
+
+Also, you must disable the "Source/Destination check" on each machine,
+because Weave will be operating with IP addresses outside of the range
+allocated by Amazon.
+
+###Using AWS-VPC mode
+
+Launch Weave Net, adding the `--awsvpc` flag:
+
+    $ weave launch --awsvpc [other hosts]
+
+You still need to supply the names or IP addresses of other hosts in
+your cluster.
```

Comment:
How can one tell whether awsvpc mode is being used by a node? Is there some indication in `weave status`? `weave status connections`?